pandas/_config/config.py

```diff
@@ -388,7 +388,7 @@ class option_context:
     Examples
     --------
 
-    >>> with option_context('display.max_rows', 10, 'display.max_columns', 5):
+    with option_context('display.max_rows', 10, 'display.max_columns', 5):
```

Comment:
What happened here?

Reply:
I thought those were some characters I had accidentally added, but now that I look at the master I realize they're supposed to be there, oops